tests/test_types.py

```diff
@@ -76,6 +85,12 @@ async def get_enums(self, string_selection: StringEnum) -> EnumTypes:
 	async def post_enums(self, body: EnumTypes) -> EnumTypes:
 		return body
 
+	async def get_actual_enums(self, e: ActualEnum) -> str:
```

Comment:
I think here it's worth returning the `e` instance wholesale as the response, as we do for other tests, to ensure it roundtrips correctly into the response data.

So the signature would be `async def get_actual_enums(self, e: ActualEnum) -> ActualEnum`, and then the same change for the post test below.

Reply:
Thinking about it, this might also require changes to the JSON response serialiser at https://github.com/returnString/govyn/blob/main/govyn/endpoint.py#L15 to unpack the `value` attribute for enum-like instances.

Reply:
all good points, fixed!